src/Angor/Client/Shared/MainLayout.razor

```diff
@@ -18,7 +18,7 @@
     <NavMenu @bind-IsCollapsed="isSidebarCollapsed" IsDarkTheme="isDarkTheme"/>
     <main class="main-content">
         <!-- Navbar -->
-        <nav class="navbar navbar-main navbar-expand-lg px-0 border-radius-xl position-sticky mt-3 z-index-sticky left-auto ms-2 me-2" id="navbarBlur">
+        <nav class="navbar navbar-main navbar-expand-lg px-0 border-radius-xl position-sticky mt-2 z-index-sticky left-auto ms-2 me-2" id="navbarBlur">
```

Comment:
Milad perhaps the message component should be in the main layout if it is a popup?